src/components/views/toasts/VerificationRequestToast.tsx

```diff
@@ -131,9 +141,10 @@ export default class VerificationRequestToast extends React.PureComponent<IProps
         let nameLabel;
         if (request.isSelfVerification) {
             if (this.state.device) {
-                nameLabel = _t("From %(deviceName)s (%(deviceId)s)", {
+                nameLabel = _t("From %(deviceName)s (%(deviceId)s) from %(IP)s", {
```

Comment:
Hmm, having two "from"s seems a bit unfortunate, perhaps the IP could go inside the parens with the device ID...?

Reply:
"from $device at $ip" maybe